app/server/middleware/check-decision.ts

```diff
@@ -9,14 +9,17 @@ const logger = Logger.getLogger('check-decision.js');
 const acceptedDecisionStates = [CONST.TRIBUNAL_VIEW_ISSUED_STATE, CONST.DECISION_REJECTED_STATE, CONST.DECISION_ACCEPTED_STATE];
 
 export function checkDecision(req: Request, res: Response, next: NextFunction) {
-  const hearing: OnlineHearing = req.session.hearing;
 
-  if (!hearing.decision) {
-    const missingDecisionError = new Error('Unable to retrieve decision from session store');
-    AppInsights.trackException(missingDecisionError);
+  const session = req.session;
+
```

Comment:
CheckDecision is functionality of JUI, we don't require. Please fix so that this function should not be called